share/messages_to_error_log.txt

```diff
@@ -11596,6 +11596,15 @@ ER_IB_SLOWRM_CREATE_DIR
 ER_IB_SLOWRM_MOVE_FILE
   eng "%s"
 
+ER_DDSE_CHANGE
+  eng "MySQL DDSE is changing from '%s' to '%s'."
+
+ER_DD_ENGINE_NOT_FOUND
+  eng "The DD_ENGINE setting for the data dictionary was not found. Starting the server using DD_ENGINE= '%u'."
+
+ER_UNSUPPORTED_DD_ENGINE
+  eng "The DD_ENGINE value '%u' found in dd_properties isn't supported DDSE."
```

Comment:
Similar problem but IMHO it wouldn't be a good idea to translate all the enum values neither. Either point the user to the source code with enum values, either add something like "supported DDSEs are InnoDB (value 1) and MyRocks (value 2)"